packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/ImageSaver.java

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera;
+
+import android.media.Image;
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+/** Saves a JPEG {@link Image} into the specified {@link File}. */
+public class ImageSaver implements Runnable {
+
+  /** The JPEG image */
+  private final Image mImage;
```

Comment:
In other classes the member fields are not prefixed with the `m` character, the same style should be used here.